### ISSUE TITLE
Ubuntu 15.04

### ISSUE THREAD

**Issue by th3m4ri0** [sockeye44/instavpn#42]
Any idea if this could be made compatible with 15.04 ? I'd be glad to help.

**Comment by th3m4ri0** [comment]
Right now the package ```openswan``` isn't 15.04 compatible, [making the installation process fail](https://pastee.org/tmjuj) : 
```
10:51:43 DEBUG: Install VPN server packages
10:51:43 DEBUG: Reading package lists...
10:51:43 DEBUG: Building dependency tree...
10:51:43 DEBUG: Reading state information...
10:51:43 DEBUG: Package openswan is not available, but is referred to by another package.
10:51:43 DEBUG: This may mean that the package is missing, has been obsoleted, or
10:51:43 DEBUG: is only available from another source
10:51:43 DEBUG: 
10:51:43 DEBUG: E: Package 'openswan' has no installation candidate
10:51:43 DEBUG: 
10:51:43 DEBUG: 
10:51:43 DEBUG: 
10:51:43 DEBUG: 
10:51:43 CRITICAL: Failed to install packages
```

**Comment by sockeye44** [comment]
As far I know, openswan will be available only for LTS (10.04, 12.04, 16.04) versions.

**Comment by th3m4ri0** [comment]
Good to know, thanks !
